docs/kits/Traceability Kit/Software Development View/Notification API/receive-quality-notification.api.mdx

```diff
@@ -0,0 +1,58 @@
of quality notification.","title":"QualityClassification"}}></SchemaItem><SchemaItem collapsible={false} name={"severity"} required={true} schemaName={"QualitySeverity"} qualifierMessage={"**Possible values:** [`MINOR`, `MAJOR`, `CRITICAL`, `LIFE-THREATENING`]"} schema={{"type":"string","enum":["MINOR","MAJOR","CRITICAL","LIFE-THREATENING"],"example":"CRITICAL","description":"The severity of the quality notification describes its criticality.","title":"QualitySeverity"}}></SchemaItem><SchemaItem collapsible={false} name={"relatedNotificationId"} required={false} schemaName={"uuid"} qualifierMessage={undefined} schema={{"type":"string","format":"uuid","example":"7895a39d-c4ef-4b75-b39f-cae8207a262f","description":"A UUIDv4 to uniquely identify a related quality notification."}}></SchemaItem><SchemaItem collapsible={false} name={"status"} required={true} schemaName={"QualityStatus"} qualifierMessage={"**Possible values:** [`CREATED`, `SENT`, `RECEIVED`, `ACKNOWLEDGED`, `ACCEPTED`, `DECLINED`, `CLOSED`]"} schema={{"type":"string","enum":["CREATED","SENT","RECEIVED","ACKNOWLEDGED","ACCEPTED","DECLINED","CLOSED"],"example":"SENT","description":"The status of the quality notification.","title":"QualityStatus"}}></SchemaItem><SchemaItem collapsible={false} name={"targetDate"} required={false} schemaName={"date-time"} qualifierMessage={undefined} schema={{"type":"string","format":"date-time","example":"2022-07-28T14:41:13.214Z","description":"The date and time when a processing of the notification is expected by the sender."}}></SchemaItem><SchemaItem collapsible={false} name={"messageId"} required={true} schemaName={"uuid"} qualifierMessage={undefined} schema={{"type":"string","format":"uuid","example":"c2801472-5f87-41a7-9a25-b0939c4e0dff","description":"A UUIDv4 to uniquely identify an individual quality notification message. In case of an initial sending of a notification the messageId has to be a newly generated UUIDv4."}}></SchemaItem></div></details></SchemaItem><SchemaItem collapsible={true} className={"schemaItem"}><details style={{}}><summary style={{}}><strong>content</strong><span style={{"opacity":"0.6"}}> object</span><strong style={{"fontSize":"var(--ifm-code-font-size)","color":"var(--openapi-required)"}}> required</strong></summary><div style={{"marginLeft":"1rem"}}><SchemaItem collapsible={false} name={"information"} required={false} schemaName={"string"} qualifierMessage={"**Possible values:** `<= 1000 characters`"} schema={{"type":"string","maxLength":1000,"example":"Gear boxes loose oil while driving."}}></SchemaItem><SchemaItem collapsible={false} name={"listOfAffectedItems"} required={true} schemaName={"string[]"} qualifierMessage={undefined} schema={{"type":"array","items":{"type":"string"},"example":["urn:uuid:57e4e3c1-a6f0-46a0-90df-1fb17cbc157d","urn:uuid:e4da568b-8cf1-4f5f-a96a-cf26265b2c72"]}}></SchemaItem></div></details></SchemaItem></ul></details></TabItem></MimeTabs><div><ApiTabs><TabItem label={"201"} value={"201"}><div>
```

Comment:
@mhellmeier i meant this section here. Its not readable with all the divs etc.

Reply:
This section isn't written manually by me. It is auto-generated from docusaurus `gen-api-docs` with the command `npm run docusaurus gen-api-docs X` as [the Wiki suggests to create API documentation](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/wiki/Creating-an-open-API-documentation).